recipes/aiozmq/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "aiozmq" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b41d589a715acc0d42b525ef7084bbcdc03c986ef5a90e212ee7f086125cf6c0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Comment:
[Looks like](https://github.com/aio-libs/aiozmq/blob/master/setup.py#L66-L70) some [`entry_points`](https://conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#python-entry-points) need to be defined.

Reply:
I've added this back (I left it out originally as it seems to mostly function as an example use of the library).